readme.md

```diff
@@ -344,6 +344,17 @@ A custom file type detector.
 
 Detectors can be added via the constructor options or by directly modifying `FileTypeParser#detectors`.
 
+### Example adding a detector
+
+```js
+import {FileTypeParser} from 'file-type';
+import {detectXml} from '@file-type/xml'; // Third party detector for XML formats
+
+const parser = new FileTypeParser({customDetectors: [detectXml]});
+const fileType = await parser.fromFile('sample.kml');
+console.log(fileType);
+```
```

Comment:
Wrong place. This should not be before the description of custom detectors. It should probably be after the `### Example writing a custom detector` section. Same in index.d.ts

Reply:
The example I added is to show how to add an existing available (plugin) detector to file-type.
I intentionally placed that **before** the section in which we explaining how to implement your own detector.

Reply:
Makes sense, but should at minimum be after `- Modifying the detection behavior of existing `FileTypeResult` types.`

Reply:
Changed quite a bit in the flow, aiming for make it more readable. Also give a better structure to allow other possible future file-type extensions.